cfme/tests/services/test_catalog.py

```diff
@@ -5,31 +5,59 @@
 import cfme.tests.configure.test_access_control as tac
 from cfme import test_requirements
 from cfme.services.catalogs.catalog import CatalogsView
+from cfme.services.catalogs.catalog import DetailsCatalogView
+from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.update import update
 
 pytestmark = [test_requirements.service, pytest.mark.tier(2)]
 
 
 @pytest.mark.rhel_testing
 @pytest.mark.sauce
-def test_catalog_crud(appliance):
+def test_catalog_crud(request, appliance):
     """
     Polarion:
         assignee: nansari
         casecomponent: Services
         initialEstimate: 1/8h
         tags: service
     """
+    # Create Catalog
     catalog_name = fauxfactory.gen_alphanumeric()
     cat = appliance.collections.catalogs.create(name=catalog_name, description='my catalog')
+    request.addfinalizer(cat.delete_if_exists)
 
     view = cat.create_view(CatalogsView)
     assert view.is_displayed
-    view.flash.assert_success_message('Catalog "{}" was saved'.format(catalog_name))
+    if appliance.version >= '5.11':
+        saved_message = f"Catalog was saved"
+    else:
+        saved_message = f'Catalog "{catalog_name}" was saved'
+    view.flash.assert_success_message(saved_message)
 
+    # Edit Catalog
+    update_descr = 'my edited description'
     with update(cat):
-        cat.description = 'my edited description'
+        cat.description = update_descr
+
+    assert cat.description == update_descr
```

Comment:
optional:
Along with `object` `attribute` check, I think we can check `cat.exist` (UI check)

Reply:
:+1: Added in the create section